src/test/java/com/znsio/e2e/runner/BrowserStackImageInjectionTest.java

```diff
@@ -0,0 +1,44 @@
+package com.znsio.e2e.runner;
+
+import io.appium.java_client.AppiumDriver;
+import io.appium.java_client.android.AndroidDriver;
+import io.appium.java_client.android.AndroidElement;
+import org.openqa.selenium.remote.DesiredCapabilities;
+import org.testng.annotations.BeforeMethod;
+import org.testng.annotations.Test;
+
+import java.net.MalformedURLException;
+import java.net.URL;
+
+import static org.testng.AssertJUnit.assertNotNull;
+
+public class BrowserStackImageInjectionTest {
+    AndroidDriver<AndroidElement> driver;
+
+    @BeforeMethod
+    public void getSetup() {
+        String userName = "poojamanna_apMVFD";
+        String accessKey = "1hYzxAksrMizydSLA2aP";
```

Comment:
Is there any alternate way instead of checking in your userName and key?

Reply:
No Anand i was not sure how to add the test without browserstack credentials ad test this flow so added the details and tested the same